docs/site/BelongsTo-relation.md

```diff
@@ -328,6 +328,13 @@ on constructor to avoid "Circular dependency" error (see
 
 ## Querying related models
 
+LoopBack 4 creates a different inclusion resolver for each relation type. Each
```

Comment:
maybe we call it `LoopBack` instead of `LoopBack 4`?

Reply:
We don't mention anything about inclusion resolver in LB3. Would it be better if I change it to `LB4`? or
> Different from LB3, LB4 creates a different inclusion resolver for each relation type to query  related models.

?